packit/copr_helper.py

```diff
@@ -187,51 +187,55 @@ def create_copr_project_if_not_exists(
 
         Raises PackitCoprException on any problems.
         """
-        logger.info(
-            f"Trying to get {owner}/{project} Copr project. "
-            "The project will be created if it does not exist.",
+        default_instructions = (
+            (
+                "You can check out the upstream project "
+                f"{self.upstream_local_project.git_url} to find out how to consume these builds. "
+                f"This copr project is created and handled by the packit project "
+                "(https://packit.dev/)."
+            )
+            if self.upstream_local_project
+            else None
+        )
+
+        delete_after_days: Optional[int] = (
+            None if preserve_project is None else -1 if preserve_project else 60
         )
+
+        logger.info(f"Creating {owner}/{project} Copr project.")
         try:
-            copr_proj = self.copr_client.project_proxy.get(
+            copr_proj = self.copr_client.project_proxy.add(
                 ownername=owner,
                 projectname=project,
-            )
-        except CoprNoResultException as ex:
-            if owner != self.configured_owner:
-                raise PackitCoprProjectException(
-                    f"Copr project {owner}/{project} not found.",
-                ) from ex
-
-            logger.info(f"Copr project '{owner}/{project}' not found. Creating new.")
-            self.create_copr_project(
                 chroots=chroots,
-                description=description,
-                instructions=instructions,
-                owner=owner,
-                project=project,
-                list_on_homepage=list_on_homepage,
-                preserve_project=preserve_project,
-                additional_packages=additional_packages,
+                description=(
+                    description
+                    or "Continuous builds initiated by packit service.\n"
+                    "For more info check out https://packit.dev/"
+                ),
+                contact="https://github.com/packit/packit/issues",
+                # don't show project on Copr homepage by default
+                unlisted_on_hp=not list_on_homepage,
+                # delete project after the specified period of time
+                delete_after_days=delete_after_days,
                 additional_repos=additional_repos,
-                targets_dict=targets_dict,
+                instructions=instructions or default_instructions,
                 module_hotfixes=module_hotfixes,
                 follow_fedora_branching=follow_fedora_branching,
+                exist_ok=True,
             )
-            return
-        except CoprRequestException as ex:
-            logger.debug(repr(ex))
-            logger.error(
-                f"We were not able to get copr project {owner}/{project}: {ex}",
+        except (CoprException, CoprRequestException) as ex:
+            error = (
+                f"Cannot create a new Copr project "
```

Comment:
```suggestion
                f"Cannot create or update the Copr project "
```

the problem can be now also in updating, or not?

Reply:
Not really, `project_proxy.add(*, exist_ok=True)` either creates a project or returns an existing one but without doing any changes to it, that happens later in `update_copr_project()` and `_update_chroot_specific_configuration()`.

Reply:
ah, ok, understood!